backend/danswer/llm/utils.py

```diff
@@ -56,16 +56,26 @@ def get_default_llm_token_encode() -> Callable[[str], Any]:
     return _LLM_TOKENIZER_ENCODE
 
 
+def tokenizer_trim_content(
+    content: str, desired_length: int, tokenizer: Encoding
+) -> str:
+    tokenizer = get_default_llm_tokenizer()
+    tokens = tokenizer.encode(content)
+    if len(tokens) > desired_length:
+        content = tokenizer.decode(tokens[:desired_length])
+    return content
+
+
 def tokenizer_trim_chunks(
```

Comment:
This isn't even used :/ should we remove it?

Reply:
I guess it's no harm to keep it, I didn't like it much cause it kind of confuses the embedding and LLM flows, but it kind of makes sense as each chunk is suppose to be 512 tokens which is determined by the embedding flow... still a lil weird